west-sbom-report.md

```diff
@@ -0,0 +1,104 @@
+> ### WARNING!
+> Some of the files does not contains license information or the license contained in them cannot be detected.<br>
+See [license details](internal_link) for list of files.
+Evaluate them manually to get license information.
+
```

Comment:
I would rather start the document with a generic short description, with a title something like Software Bill of Material report. Then have a note rather than a warning after the description text.

Reply:
Ok, so maybe: move `Summary` above note/warning and rename it to `Software Bill of Material report`

# Software Bill of Material report

> the note

# ~~Summary~~

This report was generated from the follo....